frappe/public/js/frappe/utils/utils.js

```diff
@@ -892,7 +894,14 @@ Object.assign(frappe.utils, {
 			hide_seconds: docfield.hide_seconds
 		};
 		return duration_options;
-	}
+	},
+
+	generate_route: generate_route,
```

Comment:
Why not move the implementation of these generic utilities here where they belong?

Reply:
Done @netchampfaris